source/2020-06-19-the-ember-times-issue-153.md

```diff
@@ -14,14 +14,15 @@ READMORE
 
 ---
 
-## [Section title in sentence case 🐹](section-url)
+## [Stories of becoming a developer 📖](https://dev.to/agathebadia/share-your-dev-story-51dg)
 
-<change section title emoji>
-<consider adding some bold to your paragraph>
-<please include link to external article/repo/etc in paragraph / body text, not just header title above>
+[Agathe Badia (@honeypotio)](https://github.com/Agathebadia) has written a blog post called **Share your Dev story**, this blog post introduces her new website 🎉 that was built with [ember.js](https://emberjs.com/), and it is called [Becoming a Developer](https://becoming-a-developer.netlify.app/). 
 
-<add your name to author list, top and bottom>
-<add blurb and emoji to "SOME-INTRO-HERE">
+This site is where developers can share their background, the story of how they started, interest in development, studies, first developer role, and some recommendations for anyone starting or continuing to dive into development. 💻
```

Comment:
I think it may be nice to change the article to "you" here, so that a reader may be encouraged to write a post on Agathe's website. What do you think?

```suggestion
On Becoming a Developer, you can share your background, the story of how you started, interest in coding, studies, first developer role, and some recommendations for anyone starting or continuing to dive into development. 💻
```